packages/alfa-style/src/property/combinator/any.ts

```diff
@@ -0,0 +1,44 @@
+import { Parser } from "@siteimprove/alfa-parser";
+import { Err, Result } from "@siteimprove/alfa-result";
+
+export function any<I, T, E, A extends [] = []>(
```

Comment:
Also, given that it's a tricky combinator (both to write and to understand, notably trickier than the ones we already have) it would benefits from some tests 😄